core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/EwmaModel.java

```diff
@@ -52,6 +52,22 @@ public EwmaModel(double alpha) {
     }
 
     @Override
+    public boolean canBeMinimized() {
+        return true;
+    }
+
+    @Override
+    public MovAvgModel neighboringModel() {
+        double alpha = Math.random();
```

Comment:
this isn't really a neighbouring model so much as just a random model. Maybe we should have a weighting so it picks a value of alpha based on a parameter (in the simulated annealing case we would probably pass in the temperature or factor of the temperature here)? Maybe something to think about later rather than doing this now?